include/crm/common/scheduler.h

```diff
@@ -160,6 +166,63 @@ enum pcmk_scheduler_flags {
     pcmk_sched_validate_only            = (1ULL << 27),
 };
 
+//! Implementation of pcmk_scheduler_t
+struct pe_working_set_s {
+    // Be careful about when each piece of information is available and final
+
+    xmlNode *input;                 //!< CIB XML
+    crm_time_t *now;                //!< Current time for evaluation purposes
+    char *dc_uuid;                  //!< Node ID of designated controller
+    pcmk_node_t *dc_node;           //!< Node object for DC
+    const char *stonith_action;     //!< Default fencing action
+    const char *placement_strategy; //!< Value of placement-strategy property
+    unsigned long long flags;       //!< Group of enum pcmk_scheduler_flags
```

Comment:
compat note for `uint64_t` at next opportunity?

Reply:
Might as well